client/src/components/DatasetChangingMergedProjectModal.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react'
+import { Box, Text } from 'grommet'
+import { config } from 'config'
+import { useResponsive } from 'hooks/useResponsive'
+import { Button } from 'components/Button'
+import { Modal, ModalBody } from 'components/Modal'
+import { WarningText } from 'components/WarningText'
+import { InfoText } from 'components/InfoText'
+import { Link } from 'components/Link'
+
+export const DatasetChangingMergedProjectModal = ({
+  label = 'Save Changes',
+  title = 'Changing a Merged Project',
+  disabled = false
+}) => {
+  const [showing, setShowing] = useState(false)
+  const { responsive } = useResponsive()
+
+  const handleClick = () => {
+    setShowing(true)
+  }
+
+  return (
+    <>
+      <Button
+        aria-label={label}
+        flex="grow"
+        primary
+        label={label}
+        disabled={disabled}
+        onClick={handleClick}
+      />
+      <Modal showing={showing} setShowing={setShowing}>
+        <Box
+          border={{
+            side: 'bottom',
+            color: 'border-black',
+            size: 'small'
+          }}
+          gridArea="title"
+          width="full"
+          height={{ min: 'min-content' }}
+          pad={{ left: 'medium' }}
+          margin={{ bottom: '24px' }}
+        >
+          <WarningText
+            lineBreak={false}
+            iconMargin="0"
+            iconSize="24px"
+            margin={false}
+          >
+            <Text size="xlarge">{title} </Text>
```

Comment:
Remove the extra white space here:
```suggestion
            <Text size="xlarge">{title}</Text>
```